src/main/java/org/zalando/nakadi/service/subscription/state/CleanupState.java

```diff
@@ -43,7 +42,7 @@ public void onEnter() {
             try {
                 getContext().getZkClient().close();
             } catch (final IOException e) {
-                TracingService.logErrorInSpan(cleanupScope,
+                TracingService.logErrorInSpan(getContext().getCurrentSpan(),
```

Comment:
user will definitely will not need it, it is quite internal exception, which we could track in the logs